src/components/dataset/query/QueryView.jsx

```diff
@@ -206,6 +223,8 @@ function mapStateToProps(state) {
     joinStatement: state.query.joinStatement,
     queryResults: state.query.queryResults,
     orderBy: state.query.orderBy,
+    profile: state.profiles.profile,
+    canLink: state.canLink,
```

Comment:
i think you can omit `canLink` here, only things that you're pulling from the global state need to be in `mapStateToProps`